js/everything.js

```diff
@@ -229,13 +229,15 @@ var ZenPlayer = {
                     }
                     window.sessionStorage[videoID] = resumeTime;
                 }
+                var updatedUrl;
                 if (resumeTime > 0) {
-                    var updatedUrl = that.videoUrl + "&t=" + Math.round(resumeTime);
+                    updatedUrl = that.videoUrl + "&t=" + Math.round(resumeTime);
                     $("#zen-video-title").attr("href", updatedUrl);
                 }
                 else if (resumeTime <= 0 && $("#zen-video-title").attr("href") !== that.videoUrl) {
-                    $("#zen-video-title").attr("href", that.videoUrl);
+                    updatedUrl = that.videoUrl;
```

Comment:
Not sure if we need to wrap `updatedUrl` in a `if (updatedUrl)` statement. Can't imagine any case where the variable will remain empty. What do you think?